src/domains/nodo-common/env/weu-uat/terraform.tfvars

```diff
@@ -313,6 +313,19 @@ nodo_storico_storage_account = {
   backup_retention              = 0
 }
 
+mbd_storage_account = {
+  account_kind                  = "StorageV2"
+  account_tier                  = "Standard"
+  account_replication_type      = "ZRS"
+  blob_versioning_enabled       = false
+  advanced_threat_protection    = false
+  blob_delete_retention_days    = 90
+  public_network_access_enabled = false
+  backup_enabled                = false
+  backup_retention_days         = 0
```

Comment:
the configuration should be like prod

in prod, are you sure to have retention only for 30 days?

Reply:
I will investigate about the backup retention time for prod